test/e2e-tests.sh

```diff
@@ -133,11 +133,22 @@ toggle_feature responsive-revision-gc Disabled
 # simply cannot pass.
 go_test_e2e -timeout=20m -parallel=300 ./test/scale || failed=1
 
-# Run HA tests separately as they're stopping core Knative Serving pods
-# Define short -spoofinterval to ensure frequent probing while stopping pods
+# Disable the Chaosduck for following tests.
+disable_chaosduck
```

Comment:
I'd rather harden our tests to ensure we do the right thing in the face of instability than create blind spots.